VERSION

```diff
@@ -0,0 +1 @@
+0.3.0
```

Comment:
Why is this version different than the action.yml?

Reply:
I forgot to reset it after testing. Its been updated.